scripts/exregional_make_orog.sh

```diff
@@ -344,26 +339,7 @@ fi
 #
 # Note that the orography filtering code assumes that the regional grid
 # is a GFDLgrid type of grid; it is not designed to handle ESGgrid type
-# regional grids.  If the flag "regional" in the orography filtering
-# namelist file is set to .TRUE. (which it always is will be here; see
-# below), then filtering code will first calculate a resolution (i.e.
-# number of grid points) value named res_regional for the assumed GFDLgrid
-# type regional grid using the formula
-#
-#   res_regional = res*stretch_fac*real(refine_ratio)
-#
-# Here res, stretch_fac, and refine_ratio are the values passed to the
-# code via the namelist.  res and stretch_fac are assumed to be the
-# resolution (in terms of number of grid points) and the stretch factor
-# of the (GFDLgrid type) regional grid's parent global cubed-sphere grid,
-# and refine_ratio is the ratio of the number of grid cells on the regional
-# grid to a single cell on tile 6 of the parent global grid.  After
-# calculating res_regional, the code interpolates/extrapolates between/
-# beyond a set of (currently 7) resolution values for which the four
-# filtering parameters (n_del2_weak, cd4, max_slope, peak_fac) are provided
```

Comment:
I noticed that these namelist options are still referenced in `parm/input.nml.FV3`, should they be removed there as well?

Reply:
@mkavulich - It looks like `n_del2_weak`, `cd4`, `max_slope`, and `peak_fac` are read in from the `surf_map_nml` namelist file.  The namelist file is read in from `fv3atm/GFDL_atmos_cubed_sphere/tools/fv_surf_map.F90`.  If entries are missing, then default values will be used.

If this part of the FV3 is only used for `GFDLgrid` work, then it would probably be safe to remove the entire `surf_map_nml` namelist.  If it is used for all grids, then it would be safe to leave it alone.

I'll see what parts of the code `ESGgrid` runs go into and make an assessment following that.